os/include/tinyara/binfmt/compression/compress_read.h

```diff
@@ -80,4 +80,20 @@ int compress_init(int filfd, uint16_t offset, off_t *filelen);
  ****************************************************************************/
 int compress_read(int filfd, uint16_t binary_header_size, FAR uint8_t *buffer, size_t readsize, off_t offset);
 
+/****************************************************************************
+ * Name: get_compression_header
+ *
+ * Returned Value:
+ *   Address of the compression_header
+ ****************************************************************************/
+struct s_header* get_compression_header(void);
+
+/****************************************************************************
+ * Name: get_buffers
+ *
+ * Returned Value:
+ *   Address of the buffer
+ ****************************************************************************/
+struct s_buffer* get_buffers(void);
```

Comment:
what buffers?

Reply:
According to the changes made, this function is not needed anymore and hence is removed.